returnn/torch/updater.py

```diff
@@ -9,6 +9,7 @@
 import typing
```

Comment:
```suggestion
import typing
from typing import Set
```

Reply:
Note on the (somewhat loose) import rules:

Import often used things directly, e.g. things from `typing`.

Do not import rarely used things, or also maybe ambiguous things. Instead, just import the parent module under a common name. Keep consistency. We often use `util` for `returnn.util.basic`.

Reply:
I can do that. For consistency I could also change:
```
self.optimizer = None  # type: typing.Optional[torch.optim.Optimizer]
```
to
```
self.optimizer = None  # type: Optional[torch.optim.Optimizer]
```
and
```suggestion
from typing import Set, Optional
```

Reply:
Yea ok this is a bit unrelated to this PR, but if you want, you can do that.